docs/Control/Monad/Eff/Exception.md

```diff
@@ -71,15 +71,21 @@ main = do
 catchException :: forall a eff. (Error -> Eff eff a) -> Eff (err :: EXCEPTION | eff) a -> Eff eff a
 ```
 
-Catch an exception by providing an exception handler.
+Catch an exception by providing an exception handler for the given effectful Monad.
```

Comment:
I'm not sure if this adds anything, but the use of "Monad" is probably not what you intended. The only monad in use here is `Eff (err :: EXCEPTION | eff)`, so you might say "in the `Eff` monad" instead. Note, the function arguments are _not_ monads, since only type constructors can be monads, not values.